Core/src/IfcGeometryConverter/FaceConverter.h

```diff
@@ -1255,14 +1255,15 @@ namespace OpenInfraPlatform {
 							carve::geom::vector<3> vertex =
 								carve::geom::VECTOR(point[0],
 									point[1],
-									point[2]) * length_factor;
+									point[2]) * UnitConvert()->getLengthInMeterFactor();

```

Comment:
There exists a function that translates list of Cartesian Points already available in Placement Converter. Please use that one.

Reply:
Unfortunately, I could not find a function, which could replace this part of code. Can you tell me please, where (which line) I can find this function?